src/fieldAnalyser/typeUtils.ts

```diff
@@ -25,7 +25,22 @@ export function getMostEnglobingType(
   return null;
 }
 
+export function isMultiValueFacet(obj: unknown): boolean {
+  // Handling the most obvious use case. A multi value field could also be a string with delimiter, but we do not want to assume anything...
+  // With an array, we know for sure the metadata has multiple values
+  return Array.isArray(obj);
+}
+
+export function isHierarchicalFacet(obj: unknown): boolean {
+  // TODO: check if the array values follow a isHierarchy
```

Comment:
What do you mean?

Reply:
sorry, I meant: check if the array values follow a hierarchy pattern.
e.g.: `["a", "a|b","a|b|c", ...]`

Reply:
but that is more a nice to have than anything else